src/server_manager/electron_app/package_linux_action.sh

```diff
@@ -32,4 +33,4 @@ $ROOT_DIR/src/server_manager/node_modules/.bin/electron-builder \
   --config.linux.category=Network \
   --config.artifactName='Outline-Manager.${ext}'
```

Comment:
I wonder if ` --config.artifactName="Outline-Manager.\${ext}"` is a better way to remove the warning.